src/messages.py

```diff
@@ -0,0 +1 @@
+TITLE_MESSAGE = '\n\n--------WASHU TRANSFER COURSE FINDER--------'
```

Comment:
This is also in consts.py

Reply:
let's please delete this messages.py file